pandas/io/parquet.py

```diff
@@ -18,20 +18,23 @@ def get_engine(engine: str) -> "BaseImpl":
 
     if engine == "auto":
         # try engines in this order
-        try:
-            return PyArrowImpl()
-        except ImportError:
-            pass
+        engine_impls = [PyArrowImpl, FastParquetImpl]
 
-        try:
-            return FastParquetImpl()
-        except ImportError:
-            pass
+        error_msgs = []
+        for eimpl in engine_impls:
+            try:
+                return eimpl()
+            except ImportError as ie:
```

Comment:
I think we always use `as e` or as `as err`.

Reply:
I think ```as err``` as we are trying to avoid one letter variable (*I think*)